projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+---
+stoplight-id: 73ef6e15ee3ec
+---
+
+# Project flow from test to production
+
+For the integration with the UiTdatabank Entry API, there is a flow in which you first obtain access to our test environment. 
+Access to production environment is given after your integration on test environment has been successful validated by publiq vzw.
+
+On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!
+
+> Make sure you read this guide before you start developing on our test environment!
+
+## Step 1: start on test environment
+
+You can apply for free test credentials at [Projectaanvraag](https://projectaanvraag.uitdatabank.be). 
+
+Once you've obtained your personal test credentials you can start with the development of your integration with the UiTdatabank Entry API on our test environment.
+
+## Step 2: validation by publiq vzw
+
+As soon as you have finished developing your integration with the Entry API, you send a minimum of 5 of events to our test environment. You contact us via `vragen@publiq.be` and provide us the identifiers of the created test events. We will then validate your integration as soon as possible. 
+
+In this validation process we assess the integration in two manners:
+1. Is the **created data** qualitative, relevant and valuable for public agendas such as www.uitinvlaanderen.be?
+2. Is the **integration made robust**, e.g. is it possible to update and delete content through the integration?
+
+It is important that you go through the checklist below in advance and validate your integration against it. By doing this in advance, you ensure a smooth transition to the production environment!
```

Comment:
```suggestion
It is important that you go through the checklist below in advance and validate your integration against it. By doing this in advance, you ensure a smooth transition to the production environment.
```

Super small nit but I don't think the exclamation mark is really necessary here. I checked with Flo and she says that in general it's advised to avoid exclamation marks in general copy/text. The sentence should be strong enough on its own.

It can also be removed from:

> On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!

> Make sure you read this guide before you start developing on our test environment!

> In order to create an UiTPAS event, the fields `organizer` and `priceInfo` are also mandatory!

All these sentences are clear and strong enough without the `!`.

Note: I and/or other documentation contributors may definitely be guilty of this as well, it's an easy mistake. I just happened to notice it in this particular guide because it's repeated in a couple of places. I'll make sure to also pay more attention to it in other documentation and my own writing. It can be a good guideline for our future guidelines on writing guides :)

Reply:
@corneelwille thanks for changing it. The other sentences I mentioned still have an exclamation mark. Is that on purpose (okay if you actually prefer it for some reason), or a small oversight?